### DISCUSSION TITLE
equivalent of https://github.com/getsentry/responses?

### DISCUSSION THREAD

**Discussion by cjw296** [encode/httpx#2379] [Q&A] [answered]
Is there an equivalent of the `responses` package for use with httpx?

It'd be great if that kind of functionality was built in...

**Comment by lovelydinosaur** [comment, marked as answer]
From a look at the docs, I think `respx` might be what you're looking for?

https://www.python-httpx.org/third_party_packages/#respx

**Comment by cjw296** [comment]
Perfect, thanks!
